source/python.txt

```diff
@@ -23,12 +23,15 @@ MongoDB Python Drivers
 Introduction
 ------------
 
-Welcome to the documentation site for the official MongoDB Python 
-drivers. You can add one of the following drivers to your application 
-to work with MongoDB in Python:
+Welcome to the documentation site for the official MongoDB Python
+drivers. Add one of the following drivers to your application to work with
+MongoDB in Python:
 
-- Use :doc:`PyMongo </pymongo>` for synchronous Python applications.
+- :doc:`PyMongo </pymongo>` for synchronous Python applications
```

Comment:
S: would this list be better presented as a list of the drivers/extensions with colons and descriptions? I think that would flow better than the different sentence structures in each list item right now. Might require editing the introductory sentence

Reply:
I considered that, but I think it would require adding more information into the introduction or creating a separate section for extensions, which would make the page less concise. Since PyMongoArrow is a nested list item, I don't think it needs to be parallel with the top-level ones, which are described as drivers. I think if we needed to add more Python drivers or extensions, it might be worthwhile creating a new section.